modules/client/src/main/java/org/apache/ignite/internal/client/table/ClientKeyValueView.java

```diff
@@ -94,18 +97,24 @@ public class ClientKeyValueView<K, V> extends AbstractClientView<Entry<K, V>> im
     /** {@inheritDoc} */
     @Override
     public V get(@Nullable Transaction tx, K key) {
-        return sync(getAsync(tx, key));
+        Objects.requireNonNull(key, "key");
+
+        return sync(doGet(tx, key, "getNullable"));
     }
 
     /** {@inheritDoc} */
     @Override
     public CompletableFuture<V> getAsync(@Nullable Transaction tx, K key) {
-        Objects.requireNonNull(key);
+        Objects.requireNonNull(key, "key");
+
+        return doGet(tx, key, "getNullableAsync");
```

Comment:
should it be a 
```suggestion
        return doGet(tx, key, "getAsync");
```
?

Reply:
No, it should not. It reports what method should be used instead.